parser-service/src/test/java/org/mule/parser/service/ParserServiceTestCase.java

```diff
@@ -81,9 +82,31 @@ public void oasYaml20Wrapper() {
     assertThat(wrapper.get().getType(), is(ApiType.AMF));
     assertThat(wrapper.get().getApiVendor(), is(OAS_20));
   }
+// === PARSER AUTO ===
```

Comment:
Add assertions in ParseResult.success(), because if you are adding amf errors when fallback occurs, and ParserResult.success() check if some error exists, your result will always be not success in fallback cases